probe-rs/src/probe/blackmagic/arm.rs

```diff
@@ -767,3 +768,88 @@ impl MemoryInterface<ArmError> for BlackMagicProbeMemoryInterface<'_> {
         Ok(())
     }
 }
+
+// impl ArmProbeInterface for BlackMagicProbeMemoryInterface<'_> {
```

Comment:
Please remove this, if it's unused.

Reply:
Thank you for catching that. It's from an earlier attempt.